### PR TITLE
add class for docker input

### PR DESCRIPTION
**Pull request by codec**
Simple class for https://github.com/influxdata/telegraf/tree/master/plugins/inputs/docker

### CODE REVIEW

File: templates/inputs/docker.conf.erb [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+<% end -%>
+  # Whether to report per-cpu stats or not:
+  percpu = <%= @percpu %>
+
+  # Whether to report total system cpu stats or not:
+  totalcpu = <%= @totalcpu %>
+
+<% if @tagdrop -%>
+  # Drop metrics with these tags:
+  [inputs.cpu.tagdrop]
```
Comment (chrusty) [inline templates/inputs/docker.conf.erb]: This should be "inputs.docker.tagdrop"

File: templates/inputs/docker.conf.erb [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+    <% @tagdrop.sort.each do |key, value| -%>
+    <%= key %> = <%= value %>
+    <% end -%>
+
+<% end -%>
+<% if @tagpass -%>
+  # Pass metrics with these tags:
+  [inputs.cpu.tagpass]
```
Comment (chrusty) [inline templates/inputs/docker.conf.erb]: And "inputs.docker.tagpass"

**Comment by chrusty** [comment]
Hey codec, just 2 changes then I'll merge it in

**Comment by codec** [comment]
@chrusty Good catch, thanks.

**Pull request merged**